frontend/app/src/components/accounts/blockchain/WalletImportSelection.vue

```diff
@@ -0,0 +1,97 @@
+<script setup lang="ts">
+import type { EIP1193Provider, EIP6963ProviderDetail } from '@/types';
+
+const emit = defineEmits<{
+  (e: 'import-addresses', addresses: string[]): void;
+}>();
+
+const { t } = useI18n();
+
+const { error, success, clearAll, stop, setSuccess, setError } = useClearableMessages();
+
+const loading = ref<boolean>(true);
```

Comment:
Just make sure to keep to docs open for the order of the things in the component https://docs.rotki.com/contribution-guides/vue-typescript.html#setup-script-order